build.tex

```diff
@@ -9,21 +9,18 @@ \subsubsection{Software Packaging}
 There are four types of software that are required to be handled in order to install the LSST DM Science Pipelines software packages.
 The first type are the system prerequisites that we need to have available on the system: this can include compilers, and build support tools such as \texttt{cmake}.
 The second type is Python and the set of standard Python packages.
-Although the software should be able to be built with any valid Python, for LSST development and testing we always install our own Anaconda and install a set of Python prerequisites including \texttt{numpy}, \texttt{matplotlib}, and \texttt{astropy}\cite{2018arXiv180102634T}.
-
+Theoretically the software should  build with any valid Python, but for LSST development and testing we always install our own Anaconda and
+ a set of Python prerequisites including \texttt{numpy}, \texttt{matplotlib}, and \texttt{astropy}\cite{2018arXiv180102634T}.
 The third type of packages are EUPS ``tar and patch'' packages of third-party code.
-These packages consist of a (compressed) tar file of the upstream distribution, an optional directory of patches to be applied after the source code has been extracted, and a \texttt{ups} directory containing a table file for dependency management along with a file named \texttt{eupspkg.cfg.sh} that provides instructions for overriding the default build and install behavior.
```

Comment:
I think I'd like the first half of this sentence otherwise the "this approach" sentence doesn't really make any sense. We could remove the subcommands in the next sentence instead.

Reply:
ok